_episodes/02-loop.md

```diff
@@ -158,8 +158,7 @@ of the loop body (e.g. `end for`); what is indented after the `for` statement be
 > ## What's in a name?
 >
 >
-> In the example above, the loop variable was given the name `char` as a mnemonic;
-> it is short for 'character'.
+> In the example above, the loop variable was given the name `letter` as a mnemonic;
```

Comment:
this is no longer a mnenomic, right?

Reply:
Hm... strictly speaking, I don't think `char` was a mnemonic either.  Perhaps it would be better to write "descriptive variable" in stead of mnemonic?

Reply:
I rephrased this line